system/include/emscripten/em_asm.h

```diff
@@ -214,3 +216,20 @@ void emscripten_asm_const_async_on_main_thread(const char* code, ...);
 #define EM_ASM_ARGS(code, ...) emscripten_asm_const_int(#code _EM_ASM_PREP_ARGS(__VA_ARGS__))
 #define EM_ASM_INT_V(code) EM_ASM_INT(#code)
 #define EM_ASM_DOUBLE_V(code) EM_ASM_DOUBLE(#code)
+
+#else
```

Comment:
... while not needing to remind the reader, 200 lines later, what the "else" condition here means.